src/overlays/actors/ovl_En_S_Goro/z_en_s_goro.c

```diff
@@ -74,85 +194,1374 @@ static DamageTable D_80BBF858 = {
     /* Powder Keg     */ DMG_ENTRY(1, 0x0),
 };
 
-#endif
+// D_80BBF878
+static AnimationInfoS sAnimationInfo[] = {
+    { &gGoronLyingDownIdleAnim, 1.0f, 0, -1, ANIMMODE_LOOP, 0 },        // 0
```

Comment:
```suggestion
    { &gGoronLyingDownIdleAnim, 1.0f, 0, -1, ANIMMODE_LOOP, 0 },
```
With the anim defines/enum above I don't think the index comments are needed.